examples/pie_and_polar_charts/pie_and_donut_labels.py

```diff
@@ -43,7 +43,7 @@
 
 
 def func(pct, allvals):
-    absolute = int(round(pct/100.*np.sum(allvals)))
+    absolute = round(pct/100.*np.sum(allvals))
```

Comment:
```suggestion
    absolute = round(pct / 100 * np.sum(allvals))
```

Reply:
This is just for string formatting, so the whole function can be put into a single line:

```python
return "{:.1f}%\n({:.0f} g)".format(pct, pct / 100 * np.sum(allvals))
```

It would be nicer to work with a float `0.–1.` instead of `pct` and format it using `:.1%`, but that goes beyond the scope of this method and its caller.